### PR TITLE
Feature/sc 250

### PR DESCRIPTION
**Pull request by mclaurentiu**

### CODE REVIEW

File: sem_covid/entrypoints/etl_dags/crawl_eu_action_timeline.py [hunk 1]
```diff
@@ -34,8 +33,7 @@ def extract_settings_from_module(module):
 
 def start_crawler():
     logger.info('start crawler')
-    minio = MinioAdapter(config.EU_TIMELINE_BUCKET_NAME, config.MINIO_URL, config.MINIO_ACCESS_KEY,
-                         config.MINIO_SECRET_KEY)
+    minio = StoreRegistry.minio_object_store(config.EU_TIMELINE_BUCKET_NAME)
```
Comment (costezki) [inline sem_covid/entrypoints/etl_dags/crawl_eu_action_timeline.py:36]: nice!

File: sem_covid/entrypoints/etl_dags/crawl_irish_action_timeline.py [hunk 2]
```diff
@@ -45,8 +44,7 @@ def extract_settings_from_module(module):
 
 def start_crawler():
     logger.info('start crawler')
-    minio = MinioAdapter(config.IRELAND_TIMELINE_BUCKET_NAME, config.MINIO_URL, config.MINIO_ACCESS_KEY,
-                         config.MINIO_SECRET_KEY)
+    minio = StoreRegistry.minio_object_store(config.IRELAND_TIMELINE_BUCKET_NAME)
```
Comment (costezki) [inline sem_covid/entrypoints/etl_dags/crawl_irish_action_timeline.py:47]: clean!

**Review by costezki — APPROVED** [review]
nice!

**Review by CaptainOfHacks — APPROVED** [review]
Good job!

**Pull request merged**